storybook/src/stories/Forms/TransitGateway.stories.js

```diff
@@ -74,35 +71,18 @@ export default {
 };
 
 const TransitGatewayFormStory = () => {
-  function validName(str) {
-    const regex = /^[A-z]([a-z0-9-]*[a-z0-9])?$/i;
-    if (str) return str.match(regex) !== null;
-    else return false;
-  }
-
-  function invalidCallback(stateData) {
-    return !validName(stateData.name);
-  }
-
-  function invalidTextCallback(stateData) {
-    return !validName(stateData.name)
-      ? `Name ${stateData.name} is invalid.`
-      : `Invalid Name. Must match the regular expression: /^[A-z]([a-z0-9-]*[a-z0-9])?$/i`;
-  }
-
   return (
     <TransitGatewayForm
       data={{
         global: true,
         connections: [{ tgw: "transit-gateway", vpc: "management" }],
         resource_group: "service-rg",
         name: "transit-gateway",
+        crns: [],
       }}
-      readOnlyName={true}
       vpcList={["management", "workload"]}
       resourceGroups={["service-rg", "management-rg", "workload-rg"]}
-      invalidCallback={invalidCallback}
-      invalidTextCallback={invalidTextCallback}
+      region={"us-south"}
```

Comment:
we don't need to pass region here do we ?

Reply:
we don't have to. I added it mostly to help users be aware that the tgw is specifically for that region regardless of global routing being enabled, and also to balance out the form.

I can change it to this, thoughts? 

<img width="340" alt="Screen Shot 2023-04-20 at 4 04 44 PM" src="https://user-images.githubusercontent.com/11401353/233504801-34277a96-5e88-49a1-ad8f-e4eec4d09d94.png"> <img width="250" alt="Screen Shot 2023-04-20 at 4 02 03 PM" src="https://user-images.githubusercontent.com/11401353/233504533-032d5b18-ad4e-4b6b-9db1-5810a11e0959.png">